python/tests/holy_wvg_bands.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+import meep as mp
+from meep import after_sources
+
+
+class TestHolyWvgBands(unittest.TestCase):
+
```

Comment:
Need to rename this test to `holey_wvg_bands.py`, `TestHoleyWvgBands`, etc.

Reply:
Fixed.